### ISSUE TITLE
Color contrast ratio of unselected radio button (SǪL Server Database) under type section in create new database project dialog is 1.439:1 which is less than 3:1 :A11y_SQL Azure Data Studio_Create new database project_AI4W

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#26024]
Please do not close this bug. This bug should only be closed by Trusted Tester after veriﬁcation 
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug ﬁxing faster and easier." 

### GitHub Tags:
#A11yWCAG; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Nov2024; #SQL Azure Data Studio; #WCAG1.4.3;  #Win32; #Benchmark; #FTP; #STP;  #ColorContrast;#Ai4W;

### Environment Details:
Application Name: SǪL Azure Data Studio 
Edition: Windows 11 Enterprise 
Version: 24H2 
Installed: 22-10-2024 
OS build: 26100.2033 
Experience: Windows Feature Experience Pack 1000.26100.23.0 

### Steps to Reproduce
1.Launch the application-Azure Database Project and install extension SǪL Database Projects. 
2.Tab to “Database projects” tab item present in the left menu and Hit ENTER key. 
3.Tab to "Create New Project" button or "+" button and Hit ENTER key. 
4."Create new database project” dialog will get open. 
5.Tab to the “Azure SǪL Database” and “SǪL Server Database” radio buttons under type section. 
6.Open accessibility insights for windows and check for the color contrast ratio of unselected radio button “SǪL Server Database” 
7.Observe whether color contrast ratio for the unselected radio button “SǪL Server Database” met the minimum contrast ratio of 3:1 or not. 

### Actual Results:
Color contrast ratio of unselected radio button (SǪL Server Database) under type section in create new database project dialog is 1.439:1 which is less than 3:1 

### Expected Results:
Color contrast ratio of unselected radio button (SǪL Server Database) under type section should be greater than or equal to 3:1. 
### 
### User Impact:
Low vision users will not be able to see the radio buttons if the color contrast ratio is less than 3:1.
